doc/part-1-github-ui/workflows-and-best-practices.md

```diff
@@ -1,13 +1,21 @@
-# ⭐ Workflows and best practices
+---
+description: 'PART 1: GITHUB UI'
+---
 
-{% hint style="success" %}
-PART 1: GITHUB UI
-{% endhint %}
+# ⭐ Workflows and best practices
 
 ## Learning videos
 
 The learning videos in here will talk about workflows and best practices when working with the Github UI. What you will learn will also help you to tackle the tasks that have been provided for you to do after watching the learning videos, and is a base for what you will learn later in part 2.
 
+You will learn the following from the learning videos on this page:
```

Comment:
Neatly done @samuko-things 👍🏽
But change this line to "You will learn the following in this section:"